### ISSUE TITLE
my.openhab issue with satel binding

### ISSUE THREAD

**Issue by zacofany** [openhab/openhab#3383]
Hi.
When I use my.openhab addon together with satel binding i get issues as below.
Satel is set to: satel:refresh=100

2015-11-03 17:16:29.648 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr post error
2015-11-03 17:16:29.652 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 17:18:02.857 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr poll error
2015-11-03 17:18:02.862 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 17:23:04.927 [INFO ] [.myopenhab.internal.MyOHClient] - Connected to my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 17:57:06.084 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr post error
2015-11-03 17:57:06.088 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 17:58:38.049 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr poll error
2015-11-03 17:58:38.054 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 18:00:16.197 [INFO ] [.myopenhab.internal.MyOHClient] - Connected to my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 18:33:23.217 [ERROR] [o.o.b.s.i.protocol.SatelModule] - Send/receive timeout, disconnecting module.
2015-11-03 18:33:23.218 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Closing connection to ETHM-1 module
2015-11-03 18:33:24.255 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Connecting to ETHM-1 module at 192.168.1.191:7094
2015-11-03 18:33:24.266 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - ETHM-1 module connected successfuly
2015-11-03 19:03:33.432 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr post error
2015-11-03 19:03:33.436 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 19:05:07.599 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr poll error
2015-11-03 19:05:07.604 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 19:05:35.030 [INFO ] [.myopenhab.internal.MyOHClient] - Connected to my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 19:19:48.738 [ERROR] [o.o.b.s.i.protocol.SatelModule] - Send/receive timeout, disconnecting module.
2015-11-03 19:19:48.738 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Closing connection to ETHM-1 module
2015-11-03 19:19:49.739 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Connecting to ETHM-1 module at 192.168.1.191:7094
2015-11-03 19:19:49.751 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - ETHM-1 module connected successfuly
2015-11-03 19:51:52.623 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr post error
2015-11-03 19:51:52.627 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 19:52:59.487 [INFO ] [.myopenhab.internal.MyOHClient] - Connected to my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 19:59:47.223 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr post error
2015-11-03 19:59:47.226 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 20:00:45.609 [INFO ] [.myopenhab.internal.MyOHClient] - Connected to my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 20:16:12.623 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr post error
2015-11-03 20:16:12.628 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 20:17:23.070 [ERROR] [.myopenhab.internal.MyOHClient] - Socket.IO error: com.github.nkzawa.engineio.client.EngineIOException: xhr poll error
2015-11-03 20:17:23.076 [INFO ] [.myopenhab.internal.MyOHClient] - Disconnected from my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
2015-11-03 20:38:15.673 [ERROR] [o.o.b.s.i.protocol.SatelModule] - Send/receive timeout, disconnecting module.
2015-11-03 20:38:15.673 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Closing connection to ETHM-1 module
2015-11-03 20:38:16.674 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Connecting to ETHM-1 module at 192.168.1.191:7094
2015-11-03 20:38:16.686 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - ETHM-1 module connected successfuly
2015-11-03 20:58:34.880 [INFO ] [.myopenhab.internal.MyOHClient] - Connected to my.openHAB service (UUID = 608537d5-b5d4-4ce9-a310-5552fe236451, base URL = http://localhost:8080)
##

**Comment by steve-bate** [comment]
@belovictor Any thoughts on this issue?

**Comment by mlukjani** [comment]
I have same issue, but only when I tried to connect second system to Satel via integration option / Satel binding.
When I am connecting just one I got answer:
2015-12-29 21:30:41.781 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Closing connection to ETHM-1 module
2015-12-29 21:30:42.781 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - Connecting to ETHM-1 module at 192.168.1.105:7094
2015-12-29 21:30:42.781 [INFO ] [o.o.b.s.i.protocol.Ethm1Module] - ETHM-1 module connected successfuly
2015-12-29 21:30:42.891 [INFO ] [o.o.b.s.i.protocol.SatelModule] - Connection to Integra 128-WRL LEON initialized. Version: 1.12 2013-09-10.

I am checking with Satel's support if there is a limitation via integration port or it is binding bug.
